src/main/java/seedu/ui/TextUi.java

```diff
@@ -19,12 +19,24 @@
 
     private static final String STARTUP = "Hello from \n " + LOGO;
     private static final String GREETING = "How can I help you today?";
+    private static final String HELP_MESSAGE = LINE
+            + "\tNUSModsLite accepts the following commands:-\n"
+            + "\t\t| No.| Command Syntax          |            Command Action                      |\n"
+            + "\t\t| 1. | search <module_code>    | Search module based on the given partial regex |\n"
+            + "\t\t| 2. | show <module_code>      | Display module information                     |\n"
+            + "\t\t| 3. | add <module_code>       | Add module to the Timetable                    |\n"
+            + "\t\t| 4. | delete <module_code>    | Remove module from the Timetable               |\n"
+            + "\t\t| 5. | clear                   | Remove all modules from the Timetable          |\n"
+            + "\t\t| 6. | timetable               | Display the Timetable                          |\n"
+            + "\t\t| 7. | exit                    | Exit From Program                              |\n"
+            + "\t ** Note: For details, refer to the User Guide of NUSModsLite at: "
+            + "\n\t\thttps://ay2122s1-cs2113t-w12-2.github.io/tp/UserGuide.html\n" + LINE;
```

Comment:
Could this benefit from refactoring for when more commands are added in the future?

Reply:
For table-like UI, string formatting like "%-8.10s" for example means left-justified, 8-10 characters width iirc